src/components/StatusStrip/StatusStrip.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { mount } from "enzyme";
+
+import StatusStrip from "./StatusStrip";
+
+describe("Status Strip", () => {
```

Comment:
Can you add a test that has entities with a count of 0 as I don't think they should be shown.